win32ss/user/user32/user32.spec

```
@@ -37,6 +38,8 @@
 @ stdcall ChangeDisplaySettingsW(ptr long)
 @ stdcall ChangeMenuA(long long ptr long long)
 @ stdcall ChangeMenuW(long long ptr long long)
+@ stdcall -version=0x600+ ChangeWindowMessageFilter(long long)
+@ stdcall -version=0x601+ ChangeWindowMessageFilterEx(long long long ptr)
```

```suggestion
@ stdcall -version=0x601+ ChangeWindowMessageFilterEx(ptr long long ptr)
```
because 1st parameter is a handle.